packages/addons/tools/mono/package.mk

```diff
@@ -67,8 +67,8 @@ makeinstall_target() {
 addon() {
   mkdir -p "$ADDON_BUILD/$PKG_ADDON_ID"  
 
-  make -C "$PKG_BUILD/.$HOST_NAME" install DESTDIR="$ROOT/$ADDON_BUILD/$PKG_ADDON_ID"
-  make -C "$PKG_BUILD/.$TARGET_NAME" install DESTDIR="$ROOT/$ADDON_BUILD/$PKG_ADDON_ID"
+  make -C "$PKG_BUILD/.$HOST_NAME" install DESTDIR="$ROOT/$ADDON_BUILD/$PKG_ADDON_ID" 1> /dev/null
```

Comment:
those do not belong here. should be moved to makeinstal_{host,target}(

 addon() is meant only for packaging.